src/main/java/com/github/splendor_mobile_game/websocket/handlers/reactions/EndTurn.java

```diff
@@ -38,6 +35,41 @@
  *          "userUuid": "6850e6c1-6f1d-48c6-a412-52b39225ded7"
  *      }
  * }
+ *
+ *
+ *
+ * Before changing player turn we need to check if he has performed any actions. If he didn't perform any action, but he could do so, we need to ask for confirmation of passing his turn:
+ * Example PASS_CONFIRM_RESPONSE:
+ *
+ * {
+ *     "contextId": "02442d1b-2095-4aaa-9db1-0dae99d88e03",
+ *     "type": "PASS_CONFIRM_RESPONSE",
+ *     "result": "OK",
+ *     "data": {
+ *          "userUuid": "6850e6c1-6f1d-48c6-a412-52b39225ded7"
```

Comment:
So this documentation about `PASS_CONFIRM_RESPONSE` is unnecessary as we don't use it?

Reply:
that's right. Will delete that